provisioner/clusterpy.go

```diff
@@ -748,8 +749,13 @@ func (p *clusterpyProvisioner) prepareProvision(logger *log.Entry, cluster *api.
 		}
 	}
 
+	noScheduleTaint := false
+	if v, ok := cluster.ConfigItems[decommissionNodeNoScheduleTaintKey]; ok && v == "true" {
```

Comment:
You could just test for `v == "true"` here.

Reply:
done